lib/src/consts.rs

```diff
@@ -13,9 +13,10 @@
 // limitations under the License.
 
 //! Constants for the Ethereum protocol.
+extern crate alloc;
 
+use alloc::collections::BTreeMap;
```

Comment:
We have `std` dependencies anyway, so was there any particular motivation for changing this to `alloc`?

Reply:
No reason as far as I know. I will update to import from `std`. (Done.)